lib/media/adaptation_set.js

```diff
@@ -172,6 +172,11 @@ shaka.media.AdaptationSet = class {
       return false;
     }
 
+    // We can only adapt between the same groupId.
+    if (a.groupId !== b.groupId) {
```

Comment:
If the feature is for audio streams, you shouldn't check this for video.  I think this belongs in areAudiosCompatible_().

Reply:
yes, sorry...